Data/Sys/GameSettings/GMPE01.ini

```diff
@@ -7,114 +7,137 @@
 # Add memory patches to be applied every frame here.
 
 [ActionReplay]
-# Add action replay cheats here.
-$All Mini Games Unlocked
+$All Mini Games Unlocked [Datel]
 0218FDF0 0003FFFF
 
-$Player 1 Max Coins
+$Player 1 Max Coins [Datel]
 0218FC54 000003E7
 
-$Player 1 No Coins
+$Player 1 No Coins [Datel]
 0218FC54 00000000
 
-$Player 1 Max Stars
+$Player 1 Max Stars [Datel]
 0218FC62 000003E7
 
-$Player 1 No Stars
+$Player 1 No Stars [Datel]
 0218FC62 00000000
 
-$Player 1 Always Has Magic Lamp
+$Player 1 Always Has Magic Lamp [Datel]
 0018FC3D 0000000C
 
-$Player 2 Max Coins
+$Player 2 Max Coins [Datel]
 0218FC84 000003E7
 
-$Player 2 No Coins
+$Player 2 No Coins [Datel]
 0218FC84 00000000
 
-$Player 2 Max Stars
+$Player 2 Max Stars [Datel]
 0018FC92 000003E7
 
-$Player 2 No Stars
+$Player 2 No Stars [Datel]
 0018FC92 00000000
 
-$Player 2 Always Has Magic Lamp
+$Player 2 Always Has Magic Lamp [Datel]
 0018FC6D 0000000C
 
-$Player 3 Max Coins
+$Player 3 Max Coins [Datel]
 0218FCB4 000003E7
 
-$Player 3 No Coins
+$Player 3 No Coins [Datel]
 0218FCB4 000003E7
 
-$Player 3 Max Stars
+$Player 3 Max Stars [Datel]
 0218FCC2 000003E7
 
-$Player 3 No Stars
+$Player 3 No Stars [Datel]
 0218FCC2 00000000
 
-$Player 3 Always Has Magic Lamp
+$Player 3 Always Has Magic Lamp [Datel]
 0018FC9D 0000000C
 
-$Player 4 Max Coins
+$Player 4 Max Coins [Datel]
 0218FCE4 000003E7
 
-$Player 4 No Coins
+$Player 4 No Coins [Datel]
 0218FCE4 00000000
 
-$Player 4 Max Stars
+$Player 4 Max Stars [Datel]
 0218FCF2 000003E7
 
-$Player 4 No Stars
+$Player 4 No Stars [Datel]
 0218FCF2 00000000
 
-$Player 4 Always Has Magic Lamp
+$Player 4 Always Has Magic Lamp [Datel]
 0018FCCD 0000000C
 
-$Hold Y To Roll 10
+$Hold Y To Roll 10 [Datel]
 3A1E67F0 00000800
 0416B928 41200000
 
-$Hold L To Roll 1
+$Hold L To Roll 1 [Datel]
 3A1E67F0 00000040
 0416B928 3F000000
 
-$L+X No More Moves(Use While Moving)
+$L+X No More Moves(Use While Moving) [Datel]
 8A1E67F0 00000440
 0018FC42 00000001
 0018FC72 00000001
 0018FCA2 00000001
 0018FCD2 00000001
 00000000 40000000
 
-$L+Y Infinite Moves(While Moving)
+$L+Y Infinite Moves(While Moving) [Datel]
 8A1E67F0 00000840
 0018FC42 0000000A
 0018FC72 0000000A
 0018FCA2 0000000A
 0018FCD2 0000000A
 00000000 40000000
 
-$Press Z+R To Be On Turn 20
+$Press Z+R To Be On Turn 20 [Datel]
 0A1E67F0 00000030
 0018FCFC 00000014
 
-$Press Z+L To Be On Turn 1
+$Press Z+L To Be On Turn 1 [Datel]
 0A1E67F0 00000050
 0018FCFC 00000001
 
 [Gecko]
-$Netplay Community Settings
```

Comment:
Why was this removed?

Reply:
because it just added the previous commits and no one in the netplay community toggles it